backend/tracim_backend/tests/functional/test_mail_notification.py

```diff
@@ -114,7 +164,7 @@ def test_func__create_new_content_with_notification__ok__nominal_case(
         uapi = user_api_factory.get(current_user=None)
         current_user = uapi.get_one_by_email("admin@admin.admin")
         # set admin as french, useful to verify if i18n work properly
-        current_user.lang = "fr"
+        current_user.lang = "en"
```

Comment:
Comment explain to set in french but parameter is set to english.

Reply:
you're right, that's a mistake.